src/styles/components/_navbar.scss

```diff
@@ -29,4 +21,46 @@
   @include respond-above('xl') {
     padding: 0 rem(40px);
   }
+
+  &__menu-item,
+  &__button--sign-in {
+    &:not(:last-child) {
+      margin-right: rem(16px);
+    }
+  }
+
+  &__logo {
+    display: flex;
+  }
+
+  &__menu {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  // TODO: for future implementation
+  &__menu-item {
+  }
+
+  &__link {
+    text-decoration: none;
+  }
+
+  &__button {
+    &--sign-in {
+      // TODO: for future implementation
+    }
+
+    &--sign-up {
+      // TODO: for future implementation
+    }
```

Comment:
I want you to check with Github's navbar, they use's `<a>` for both. What do you think?